docs/guides/qiskit-code-assistant-vscode.mdx

```diff
@@ -58,36 +60,62 @@ The first time you use the `granite-8b-qiskit` model, a modal opens listing some
 
 ![The window that opens upon first use contains restrictions tha must be accepted.](/images/guides/qiskit-code-assistant/eula.png)
 
-### Generate Code
-After accepting the EULA, you can open a new file and start coding. To get a code suggestion, type `Ctrl` + `.` as shown:
+### Generate code
+
+While you develop your code using Qiskit, you can ask to the Qiskit Code Assistant to help you. In general, the assistant suggests better code as response to Python comments or docstrings, but you can use the assistant anywhere in your file.
+
+To get a code suggestion, type a prompt, then `Ctrl`+`.`.  There are two types of prompts you can use:
+
+  - Enter partial code and get a suggestion to finish the code.  Example:
+    ```python
+    from qiskit.circuit import QuantumCircuit
+    from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
+    from qiskit_ibm_runtime import SamplerV2 as Sampler
+    from qiskit_ibm_runtime.fake_provider import FakeManilaV2
+
+    # Bell Circuit
+    qc = QuantumCircuit(2)
+    qc.h(0)
+    qc.cx(0, 1)
+
+    qc.mea # Type Ctrl + . for a code suggestion.
+    ```
+  - Type a pound (#) sign, then type your prompt.  Example: `# Give me a random circuit of 5 qubits and a depth of 4.`
 
 ![A code suggestion is shown.  The suggested code is in grey.](/images/guides/qiskit-code-assistant/vscode-inline-suggestion.png)
```

Comment:
```suggestion
![A code suggestion is shown, and is greyed out.](/images/guides/qiskit-code-assistant/vscode-inline-suggestion.png)
```

Reply:
Not sure if this has improved things at all, but for accessibility reasons I changed it to "greyed out" to avoid relying on a specific color. (I don't think I accomplished my goal 🤣 ) @beckykd are you aware of an accessibility-friendly way to describe something being greyed out?